_posts/2020-11-05-data-visualization-in-R.md

```diff
@@ -0,0 +1,141 @@
+
+Before getting started, make sure that you have [installed](https://ggplot2.tidyverse.org/index.html) "tidyverse" and "ggplot" or "ggplot2". If you are using GitHub you can also choose to install "devtools".
+
+# Types of figures  
+## Bar charts   
+Bar charts are great for representing categorical data. For example, organizing your results into groups by sample type, sample origin, or experimental results. Bar charts represent average counts and are useful in showing comparisons among discrete variables as opposed to continuous variables.
+### Example  
+You can choose to represent the data with either horizontal or vertical columns depending on the range size. I recommend that for large datasets with numerous independent variables a horizontal bar chart is used and for data with less than ten discrete variables, a column bar chart is used.
+
+![Standard or column bar chart](https://ggplot2.tidyverse.org/reference/geom_bar-2.png)
+
+![Flipped y-axis sequential bar chart](https://ggplot2.tidyverse.org/reference/geom_bar-3.png)
+
+## Line Graphs   
+Line graphs a traditionally a good representation of continuous data, for example, results over time. Another common use for a line graph would be to plot a trend line or standard curve.  
+### Example  
+Unlike bar charts where the independent variable may go on either the x-axis or y-axis, line graphs exclusively place the independent variable on the x-axis.  
+![line graph](https://ggplot2.tidyverse.org/reference/geom_path-1.png)
+
+## Box Plots  
+Box plots are very useful when a scientists is trying to demonstrate distributions. These plots show the density distribution, the mean, max, and minimum. Additionally, customization allows the user to exclude outliers in these calculations while still including them on the plot. A common alternative to a box plot is a violin plot.  
+### Example
+Box plots and violin plots can be customized to include or exclude outliers when representing the density distribution.
+![Box plot showing outliers](https://ggplot2.tidyverse.org/reference/geom_boxplot-1.png)
+
+## Scatter/Point Plots  
+Scatter plots are optimal for data that you have many data points for which an average does not make sense but you would like to highlight specific grouping or trends.
+### Example
+![Classic scatter plot with categories](https://ggplot2.tidyverse.org/reference/geom_point-2.png)
+
+# Using  `ggplot`  
+`ggplot` is a data visualization system from tidyverse that takes input data sets and creates customizable graphics. For information beyond this post and user guides, github provides a [ggplot overview](https://ggplot2.tidyverse.org/index.html). Prior to running any script you will have to load the library for  `ggplot` or `ggplot2`.  Note that this is different than installing the packages.
+
+To do this: ```library(ggplot2)```
+
+## Writing a script and data input
+The general format of a ggplot input is as follows:
+```
+{r name_of_plot}
```

Comment:
Don't need this actually -- this is just for Rmarkdown chunks, so I would remove it.